_posts/fr/newsletters/2023-05-17-newsletter.md

```diff
@@ -0,0 +1,263 @@
+
+    Ces deux techniques permettent de passer de deux transactions au moins
+    à une seule, ce qui permet d'économiser une quantité
+    considérable d'espace dans les blocs. Lorsque le [traitement par
+    lots][topic payment batching] est utilisé, les économies d'espace
+    peuvent être encore plus importantes. Mieux encore, du point de vue
+    du destinataire initial (par exemple Bob), le payeur initial (par
+    exemple Alice) peut payer tout ou partie des frais. Au-delà des
+    économies d'espace et de frais, le fait de retirer les transactions
+    de la chaîne de blocs et de combiner des opérations telles que la
+    réception et la dépense rend beaucoup plus difficile pour les
+    organisations de surveillance de la chaîne de blocs de tracer de
+    manière fiable le flux de fonds.
+
+    À l'heure où nous écrivons ces lignes, le message n'a fait l'objet
+    d'aucune discussion sur la liste de diffusion.
+
+- **Résumé de la réunion en présentiel des développeurs de Bitcoin Core :** Plusieurs
+  développeurs travaillant sur Bitcoin Core se sont récemment réunis pour
+  discuter de certains aspects du projet. Les notes de plusieurs discussions
+  de cette réunion ont été publiées. Les sujets abordés comprenaient [fuzz
+  testing][], [assumeUTXO][], [ASMap][], [silent payments][], [libbitcoinkernel][],
+  [refactoring (or not)][], et [package relay][]. Deux autres sujets méritant
+  une attention particulière ont également été abordés :
+
+
+    - [Mempool clustering][] résume une suggestion de refonte significative
+      de la manière dont les transactions et leurs métadonnées sont stockées
+      dans le mempool de Bitcoin Core. Les notes décrivent un certain nombre
+      de problèmes avec la conception actuelle, fournissent une vue d'ensemble
+      de la nouvelle conception, et suggèrent certains des défis et des compromis
+      impliqués. Une [description][bitcoin core #27677] de la conception et une
+      copie des [diapositives][mempool slides] de la présentation ont été
+      publiées ultérieurement.
+
+    - La [méta discussion sur le projet][] résume une discussion variée sur les
+      objectifs du projet et la façon de les atteindre malgré de nombreux défis,
+      internes et externes. Certaines de ces discussions ont déjà conduit à des
+      changements expérimentaux dans la gestion du projet, tels qu'une approche
+      plus centrée sur le projet pour la prochaine version majeure après la version 25.
+
+## En attente de confirmation #1 : pourquoi avons-nous un mempool ?
+
+_Le premier segment d'une série hebdomadaire limitée sur le relais de transaction,
+l'inclusion dans le mempool et la sélection des transactions minières--- y compris
+pourquoi Bitcoin Core a une politique plus restrictive que celle permise par le
+consensus et comment les portefeuilles peuvent utiliser cette politique de la
+manière la plus efficace._
+
+{% include specials/policy/en/01-why-mempool.md %}
```

Comment:
@Copinmalin Did you plan on translating this section as well?

Reply:
@Copinmalin when you are done, you can change this link accordingly

Reply:
OK, done !